indexer/entity/src/dex_mean_price.rs

```diff
@@ -0,0 +1,66 @@
+use sea_orm::entity::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel, Deserialize, Serialize)]
+#[sea_orm(table_name = "DexMeanPrice")]
+pub struct Model {
+    #[sea_orm(primary_key, column_type = "BigInteger")]
+    pub id: i64,
+    #[sea_orm(column_type = "BigInteger")]
+    pub tx_id: i64,
+    #[sea_orm(column_type = "BigInteger")]
+    pub address_id: i64,
+    pub dex: i32,
+    #[sea_orm(column_type = "BigInteger", nullable)]
+    pub asset1_id: Option<i64>,
+    #[sea_orm(column_type = "BigInteger", nullable)]
+    pub asset2_id: Option<i64>,
+    #[sea_orm(column_type = "BigUnsigned")]
+    pub amount1: u64,
+    #[sea_orm(column_type = "BigUnsigned")]
+    pub amount2: u64,
+}
```

Comment:
What is the benefit of representing the mean price as a separate table with separate logic instead of a materialized view? Maybe there is a good reason for this, and if so we should write it down somewhere

Reply:
We split tables because data represents different approaches. The mean price is the theoretical price for an asset, swap is the real price.
But there is no problem with merging tables

Reply:
Right, but why can't the theoretical price be a view instead merging tables or having separate tables? It'd be nice to outline the tradeoffs and which one is picked

Reply:
I have merged tables and crate views for the webserver.

Reply:
view removed. Everything is routed from joined tables